backend/api/openapi.yaml

```diff
@@ -408,6 +409,46 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /videos/{videoId}/watch:
```

Comment:
I'd like some more ideas on this one, just in case we want to make a streaming API or something more substantial.